test/forms/event1/BenefitsPaidEarlyFormProviderSpec.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.event1
+
+import forms.behaviours.StringFieldBehaviours
+import forms.event1.member.BenefitsPaidEarlyFormProvider
+import play.api.data.FormError
+
+class BenefitsPaidEarlyFormProviderSpec extends StringFieldBehaviours {
+
+  private val requiredKey = "benefitsPaidEarly.error.required"
+  private val lengthKey = "benefitsPaidEarly.error.length"
+  private val maxLength = 150
+
+  private val form = new BenefitsPaidEarlyFormProvider()()
+
+  ".value" - {
+
+    val fieldName = "value"
+
+    behave like fieldThatBindsValidData(
+      form,
+      fieldName,
+      stringsWithMaxLength(maxLength)
+    )
+
+    behave like fieldWithMaxLength(
+      form,
+      fieldName,
+      maxLength = maxLength,
+      lengthError = FormError(fieldName, lengthKey, Seq(maxLength))
+    )
+
+    behave like mandatoryField(
```

Comment:
as far as i know, this is not mandatory. you should test this empty string instead.

Reply:
I can't see an appropriate behaviour for empty strings as this is an optional field so I've removed this behaviour test in the next push.

Reply:
"bind empty data" in {
      val result = form.bind(Map(fieldName -> "")).apply(fieldName)
      result.value.value mustBe ""
      result.errors mustBe empty
    }